api/src/main/java/org/eclipse/microprofile/jwt/builder/JwtEncryption.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2016-2017 Contributors to the Eclipse Foundation
+ *
+ *  See the NOTICE file(s) distributed with this work for additional
+ *  information regarding copyright ownership.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  You may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package org.eclipse.microprofile.jwt.builder;
+
+import java.security.PublicKey;
+
+import javax.crypto.SecretKey;
+
+/**
+ * JWT JsonWebEncryption.
+ */
+public interface JwtEncryption {
+
+    /**
+     * Encrypt the claims or inner JWT with {@link PublicKey}.
+     * 'RSA-OAEP-256' key and 'A256GCM' content encryption algorithms will be used
```

Comment:
RSA-OAEP-256 is only for using the RSA Public Key, A256GCM only for an AES key (SecretKey)

Reply:
That is what that comment implies. the former is for the encrypting the content key, the latter is for this content key to encrypt the actual content